src/main/resources/db/migration/courtcase/V2177__remove_duplicate_hearing_outcomes_from_hearing_outcome_table.sql

```diff
@@ -0,0 +1,15 @@
+BEGIN;
+-- This and the following migration are to tidy up bad data that has been caused by the process unresulted case cron job
+-- the cron used an incorrect method to checking if the hearing had been resulted
+
+-- keep the first hearing outcome
+DELETE FROM hearing_outcome dup_ho
+USING hearing_outcome dist_ho
+WHERE dup_ho.id > dist_ho.id
+  AND dup_ho.fk_hearing_defendant_id  = dist_ho.fk_hearing_defendant_id;
```

Comment:
for extra caution we can check that the type is 'NO_OUTCOME' because those are the duplicates being created by the job.

where .... AND xyz = 'NO_OUTCOME'